.github/workflows/tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Run Tests on Pull Request
+
+on:
+  pull_request:
+    branches:
+      - dev
+      - staging
+      - main
+
+jobs:
+  test-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16.x"
+      - name: Copy Proto Files
```

Comment:
I think it would be a better idea to create a tmp dir and copy the files you need there and then install. this way you don't need to create and remove `edgepi_rpc_protos` under the repo folder, which potentially can cause issues we fail to remove. 

let's change all the other workflows to follow the same procedure.

Reply:
This is what's it's doing already. It's creating the temp directory edgepi_rpc_protos inside javascript_rpc